src/fs/glob.go

```diff
@@ -174,10 +240,26 @@ func (globber *Globber) walkDir(rootPath string) (walkedDir, error) {
 	return dir, nil
 }
 
+// globPrefix extracts the fixed prefix part from a glob expression.
+// i.e. "src/fs/**/*.go" -> ["src", "fs"], ["**", "/*.go"]
```

Comment:
nit: The last bit tripped me. I think you meant just `// i.e. "src/fs/**/*.go" -> ["src", "fs"]`?

Reply:
Ah yes, this changed a bit over time, I think earlier it did used to return two slices.